update_conda_lock/action.yml

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier:	ISC
+
+name: 'Conda Lock Updating Action'
+description: 'Action Creates a Pull Request with Conda Lock Update'
+
+inputs:
+  commit_message:
+    description: 'Message of the bump commit'
+  user_name:
+    description: 'Name of the user creating commit'
+  user_email:
+    description: 'Email address of the user creating commit'
+  branch_name_core:
+    description: "Name of the pushed branch; it will be suffixed with GH Action's Run ID"
+  environment_file:
+    description: 'Path of the `environment.yml` file'
+  conda_lock_file:
+    description: 'Path of the Conda Lock file (needs to have txt/yml/yaml extension)'
+  pr_title_core:
+    description: 'Title of the Pull Request; it will be suffixed with UTC time and date'
+  gh_access_token:
+    description: 'GitHub Access Token; use Personal Access Token to trigger workflows when issuing the Pull Request.'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+  - shell: bash
+    run:   $GITHUB_ACTION_PATH/check_prepare_env.sh
+
+  - shell: bash
+    run: |
```

Comment:
This would probably be better as a Python script?

Reply:
I'm not sure if you meant it this way but I can't make it a separate script since it uses GH Action inputs so it has to be in the `action.yml` file.

If you just thought about converting this step to Python and leaving it in `action.yml` then I'm not sure what are the advantages?